ntoskrnl/config/cmsysini.c

```diff
@@ -1357,7 +1357,7 @@ VOID
 NTAPI
 CmpInitializeHiveList(VOID)
 {
-    WCHAR FileBuffer[MAX_PATH], RegBuffer[MAX_PATH];
+    WCHAR FileBuffer[64], RegBuffer[64];
```

Comment:
What are these arbitrary-looking "64" values?

Reply:
64 is exactly as arbitrary as MAX_PATH, and both are large enough to hold these values.

Reply:
The longest strings that should need to fit in here:
`RegBuffer`: "\Registry\Machine\Hardware" = 26
`FileBuffer`: "\SystemRoot\System32\Config\Hardware"  = 36
However using these strings to determine the size would make `CmpMachineHiveList` kind of pointless.